Doc/library/functions.rst

```diff
@@ -209,19 +209,18 @@ are always available.  They are listed here in alphabetical order.
           @classmethod
           def f(cls, arg1, arg2, ...): ...
 
-   The ``@classmethod`` form is a function :term:`decorator` -- see the description
-   of function definitions in :ref:`function` for details.
+   The ``@classmethod`` form is a function :term:`decorator` -- see
+   :ref:`function` for details.
 
-   It can be called either on the class (such as ``C.f()``) or on an instance (such
-   as ``C().f()``).  The instance is ignored except for its class. If a class
-   method is called for a derived class, the derived class object is passed as the
-   implied first argument.
+   A class method can be called either on the class (such as ``C.f()``) or on an
+   instance (such as ``C().f()``).  The instance is ignored except for its
+   class.  If a class method is called for a derived class, the derived class
+   object is passed as the implied first argument.
```

Comment:
Note: to minimize diff churn (to keep `git log`, `git annotate` etc. most useful), sometimes it’s fine to make one line a little too long (or very short) to avoid reflowing the entire paragraph.  (I mean: keep `instance (such` on the preceding line.)

Most text tools are still line-based, and making changes easier to read still has some value.

Reply:
Alright :) I'll update the PR so the diff is easier to read.

Reply:
Done!